app/Bootstrap.php

```diff
@@ -83,7 +83,7 @@ function ( Request $request, Application $app ) {
 
 		$app->error( function ( AccessDeniedException $e ) use ( $ffFactory ) {
 			return new Response(
-				$ffFactory->newAccessDeniedHtmlPresenter()->present( $e ),
+				$ffFactory->newAccessDeniedHtmlPresenter()->present( $e->getMessage() ),
```

Comment:
The access  denied presenter is still an InternalErrorHmlPresenter, which expects and exception for `present`.